heron/scheduler-core/src/java/com/twitter/heron/scheduler/SubmitterMain.java

```diff
@@ -315,18 +315,17 @@ public static void main(String[] args) throws Exception {
                 topologyPackage, topologyBinaryFile, topologyDefnFile, topology))
             .build());
 
-    LOG.fine("Static config loaded successfully ");
+    LOG.fine("Static config loaded successfully");
     LOG.fine(config.toString());
 
     SubmitterMain submitterMain = new SubmitterMain(config, topology);
-    boolean isSuccessful = submitterMain.submitTopology();
-
-    // Log the result and exit
-    if (!isSuccessful) {
+    try {
+      submitterMain.submitTopology();
+    } catch (TopologySubmissionException e) {
+      System.out.println(e.getMessage());
```

Comment:
This pattern of logging to stdout and then throwing RuntimeException looks odd to the reader so best to clarify why we do both in a comment.

I suspect we'll want to do similar in other places, like RuntimeManagerMain, so at some point we'll want this logic in a utility method. At that point, we might also want a common super-class above TopologySubmissionException, TopologyUpdateException, TopologyRestartException, etc.